src/app/shared/components/info/info.component.html

```diff
@@ -137,3 +143,106 @@ <h3 class="text--center text--blue">Special Thanks</h3>
     </div>
   </div>
 </ng-template>
+
+<ng-template #ideaTemplate>
+  <div class="our-team-container">
```

Comment:
Please change this class name and subsequent class names to `idea-container` or something. also, add child divs only if needed.

Reply:
@subbu3012 - you mean to use hyphenated instead of camelCase?

Reply:
@vraravam no. the container's class name should be `idea-container` instead of `our-team-container` since the template is `ideaTemplate`.

Reply:
Done. I was following same layout as other containers in same Info component.